src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+
+/** 
+* Main class of the programme
+*/ 
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Duke() throws FileNotFoundException {
+        ui = new Ui();
+        this.storage = new Storage("../duke.txt");
```

Comment:
Might be better to be more consistent with use of `this.`, for example `this.ui = new Ui();` or leave out `this.` for all three fields.